### PR TITLE
Finish new funding

### PR DESCRIPTION
**Pull request by mmazanec22**
- Use nav element for secondary nav
- Get enrollment update to look like mockup mostly
- Add pencil svg and use it for edit buttons
- Convert directional link to text with icon b/c we have more use cases for that and buttons can take JSX

### CODE REVIEW

File: src/Hedwig/ClientApp/src/components/TextWithIcon/TextWithIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+			title?: string | undefined;
+		}
+	>;
+	text: string;
+	direction?: 'left' | 'right' | 'up' | 'down';
+	iconSide?: 'left' | 'right';
+	className?: string;
+};
+
+// AKA Old Iconsides
```
Comment (jlhogan) [inline src/Hedwig/ClientApp/src/components/TextWithIcon/TextWithIcon.tsx:15]: iconsides?

File: src/Hedwig/ClientApp/src/containers/Enrollment/_sections/EnrollmentFunding_NEW/UpdateForm/EnrollmentFunding/index.tsx [hunk 2]
```diff
@@ -44,22 +49,21 @@ export const EnrollmentFundingForm = ({ enrollment, siteId }: SectionProps) => {
...
+		include: ['enrollments', 'fundings', 'funding_spaces'],
+	};
 
-	const { data: organization, error: organizationError, loading: organizationLoading } = useApi<Organization>(
-		(api) => api.apiOrganizationsIdGet(params),
-		{
-			skip: !user,
-		}
-	);
+	const { data: organization, error: organizationError, loading: organizationLoading } = useApi<
```
Comment (jlhogan) [inline src/Hedwig/ClientApp/src/containers/Enrollment/_sections/EnrollmentFunding_NEW/UpdateForm/EnrollmentFunding/index.tsx:55]: this is what the formatter did?? i feel like this is _not_ easy to look at! but oh well

File: src/Hedwig/ClientApp/src/components/TextWithIcon/TextWithIcon.tsx (hunk 1 above)
Comment (mmazanec22) [reply src/Hedwig/ClientApp/src/components/TextWithIcon/TextWithIcon.tsx:15]: Lol sorry just a [stupid joke](https://en.wikipedia.org/wiki/Old_Ironsides_(poem))

Comment (jlhogan) [reply src/Hedwig/ClientApp/src/components/TextWithIcon/TextWithIcon.tsx:15]: i would go ahead and say that joke was a SMART joke and that is why it went over my head :upside_down_face:

**Pull request merged**